guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -79,6 +88,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::foreman-el,katello,satellite[]
+// Upgrading Project from Enterprise Linux 7 to Enterprise Linux 8 using Leapp
+include::common/modules/proc_upgrading-project-in-place-using-leapp.adoc[leveloffset=+1]
+endif::[]
+
 // == Migrating Project
 ifdef::foreman-el,katello,satellite[]
```

Comment:
Again, what about foreman-deb and orcharhino?
cc @maximiliankolb

Reply:
That technically would work for foreman-deb, but I don't think the current wording works for non-EnterpriseLinux systems

Reply:
So an upgrade to EL8 is not required for foreman-deb? Or we need to figure something else out for the intro.

Reply:
Correct. foreman-deb runs on Debian, not on Enterprise Linux.